src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  --active-color--: #00acdc;
+}
+
+* {
+  border: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  box-sizing: border-box;
+  background-color: #d3d3d3;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
```

Comment:
Move `all` `reset type` styles to the top of the file and apply them to tag name selector